src/main/webapp/js/components/interface/controlPanel/controlpanel.js

```diff
@@ -152,11 +152,23 @@ define(function (require) {
             return inputStr.replace(/\$projectId\$/gi, projectId).replace(/\$experimentId\$/gi, experimentId);
         },
 
+        attachTooltip: function (content) {
+            $('.control-panel-parameter-input').uitooltip({
+                position: {my: "right+50", at: "left-100"},
+                tooltipClass: "tooltip-container",
+                content: content
+            });
+        },
+
         componentDidMount: function () {
             // listen to experiment status change and trigger a re-render to refresh input / read-only status
             GEPPETTO.on(GEPPETTO.Events.Experiment_completed, this.refresh, this);
             GEPPETTO.on(GEPPETTO.Events.Experiment_running, this.refresh, this);
             GEPPETTO.on(GEPPETTO.Events.Experiment_failed, this.refresh, this);
+            if (!GEPPETTO.UserController.hasPersistence())
```

Comment:
@mattearnshaw should this check first that the deployment is using a persistence bundle?

Reply:
@tarelli good point. `GEPPETTO.UserController.hasWritePermissions` ?

Reply:
@mattearnshaw mmm I think that method you are already using does what I'm saying but not what you thought it was doing... I think it should be
`if (GEPPETTO.UserController.hasPersistence() && !window.Project.persisted)`
but I haven't tested it.

Reply:
@tarelli thanks, fixed